hwy/detect_compiler_arch.h

```diff
@@ -147,7 +147,7 @@
 #define HWY_ARCH_ARM_A64 0
 #endif
 
-#if defined(__arm__) || defined(_M_ARM)
+#if (defined(__ARM_ARCH) && __ARM_ARCH == 7) || (defined(_M_ARM) && _M_ARM == 7)
```

Comment:
I was under impression that windows required v8... so testing equality is a bit strong right ?

Reply:
Thanks for checking, the documentation does mention 7: https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170
(For v8 they have another macro)